Don't remove sounds on dist-clean, upstream tarballs come with
one of them.

 -- Faidon Liambotis <paravoid@debian.org>

Currently no upstream issue. But anyway, we should revisit it when we decide
on packaging of sound files.

--- a/Makefile
+++ b/Makefile
@@ -459,7 +459,6 @@ dist-clean: distclean
 
 distclean: $(SUBDIRS_DIST_CLEAN) _clean
 	@$(MAKE) -C menuselect dist-clean
-	@$(MAKE) -C sounds dist-clean
 	rm -f menuselect.makeopts makeopts menuselect-tree menuselect.makedeps
 	rm -f makeopts.embed_rules
 	rm -f config.log config.status config.cache
@@ -469,6 +468,10 @@ distclean: $(SUBDIRS_DIST_CLEAN) _clean
 	rm -rf doc/api
 	rm -f build_tools/menuselect-deps
 
+# tarballs distributed by Digium include sounds
+all-clean: distclean
+	@$(MAKE) -C sounds dist-clean
+
 datafiles: _all
 	if [ `$(ID) -u` = 0 ]; then \
 		CFLAGS="$(_ASTCFLAGS) $(ASTCFLAGS)" build_tools/mkpkgconfig $(DESTDIR)/usr/lib/pkgconfig; \
